common/debug.c

```diff
@@ -206,3 +207,9 @@ print_string(const char *s)
 	while ((c = *s++))
 		console_putc(c);
 }
+
+void
+run_tests()
```

Comment:
This doesn't match the prototype for this function `()` ≠ `(void)`. Also, this function belongs in `common/test.c`

Reply:
done